docs/docs/genai/eval-monitor/scorers/llm-judge/migration-guide.mdx

```diff
@@ -0,0 +1,570 @@
+        inputs={"request": "What is the capital of France?"},
+        expectations={
+            "context": ["Paris is the capital of France.", "France is in Europe."],
+            "expected_facts": ["Paris is the capital"],
+            "expected_response": "Paris",
+        },
+    )
+    ```
+
+  </TabItem>
+</Tabs>
+
+### Migrating `is_grounded`
+
+<Tabs>
+  <TabItem value="legacy" label="Legacy Approach" default>
+    ```python
+    from mlflow.genai.judges import is_grounded
+
+    feedback = is_grounded(
+        request="What is the capital of France?",
+        response="Paris is the capital of France.",
+        context={"content": "Paris is the capital city of France."},
+        model="openai:/gpt-4",
+    )
+    ```
+
+  </TabItem>
+  <TabItem value="new" label="New Approach">
+    ```python
+    from mlflow.genai.judges import make_judge
+
+    groundedness_judge = make_judge(
+        name="groundedness",
+        instructions="""
+        Check if the response {{response}} is grounded in the provided context {{context}}.
+        The response should only contain information that can be verified from the context.
+        Answer yes if grounded, no if not grounded.
+        """,
+        model="openai:/gpt-4",
+    )
+
+    feedback = groundedness_judge(
+        inputs={"request": "What is the capital of France?"},
+        outputs={"response": "Paris is the capital of France."},
+        expectations={"context": "Paris is the capital city of France."},
+    )
+    ```
+
+  </TabItem>
+</Tabs>
+
+### Migrating `meets_guidelines`
+
+<Tabs>
+  <TabItem value="legacy" label="Legacy Approach" default>
+    ```python
+    from mlflow.genai.judges import meets_guidelines
+
+    feedback = meets_guidelines(
+        guidelines=["Be polite and respectful", "Provide accurate information"],
+        context={"response": "Hello! Paris is the capital of France."},
+        model="openai:/gpt-4",
+    )
+    ```
+
+  </TabItem>
+  <TabItem value="new" label="New Approach">
+    ```python
+    from mlflow.genai.judges import make_judge
+
+    guidelines_judge = make_judge(
+        name="guidelines_compliance",
+        instructions="""
+        Check if the response {{response}} meets the following guidelines:
+        {{guidelines}}
+
+        Answer yes if all guidelines are met, no if any are violated.
+        """,
+        model="openai:/gpt-4",
+    )
+
+    feedback = guidelines_judge(
+        outputs={"response": "Hello! Paris is the capital of France."},
+        expectations={
+            "guidelines": ["Be polite and respectful", "Provide accurate information"]
+        },
+    )
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Migration in MLflow Evaluate Context
+
+<Tabs>
+  <TabItem value="legacy" label="Legacy Evaluation Approach" default>
+    ```python
+    from mlflow.genai.judges import is_correct, is_grounded
+    import mlflow
+
+    # Legacy judges had to be wrapped or used indirectly
+    def evaluate_with_legacy():
```

Comment:
I don't think this is the wrapping that we proposed. The guideance IIRC was to create a `@scorer` method that would call into these judges.